proto/akash/escrow/v1beta1/types.proto

```diff
@@ -53,19 +53,31 @@ message Account {
     (gogoproto.moretags) = "yaml:\"state\""
   ];
 
-  cosmos.base.v1beta1.Coin balance   = 4 [
+  cosmos.base.v1beta1.Coin credits_balance   = 4 [
```

Comment:
let's use this for tracking balance as-is. Let's introduce a new tag `remaining_credits` or something extra